oddbird/Metecho
# Tour steps

## Cute animal pic
![image](https://user-images.githubusercontent.com/41588129/121398937-321c0680-c956-11eb-94dc-5f4d47d7af92.png)

_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._



## Link to Trello card (if applicable)
https://trello.com/c/lS2bwVfM/279-as-a-user-i-want-to-quickly-see-which-self-guided-tour-steps-i-have-already-viewed
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._



## Description
_The commit messages say what you did; this should explain why and/or how._
- [ x] Description is in Trello card



## Steps to test/reproduce
- Turn on the Self-guided Tour
- Click on the Tour popover buttons
- Viewed popover step buttons should be visibly different from the ones not yet seen
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._



## Show me
Screenshots to come . . .
_Provide screenshots/animated gifs/videos if necessary._
